contrib/pipglr/README.md

```diff
@@ -0,0 +1,107 @@
+## Overview
+
+This container image is built daily from this `Containerfile`, and
+made available as:
+
+* quay.io/FIXME/FIXME
+
+It's purpose is to provide an easy method to execute a GitLab runner,
+to service CI/CD jobs for groups and/or repositories on
+[gitlab.com](https://gitlab.com).  It comes pre-configured to utilize
+the gitlab-runner app to execute with rootless podman containers,
+nested inside a rootless podman container.
+
+This is intended to provide multiple additional layers of security
+for the host, when running potentially arbitrary CI/CD code.  Though,
+the ultimate responsibility still rests with the end-user to review
+the setup and configuration relative to their own situation/environment.
+
+### Quickstart
+
+Several labels are set on the built image or manifest list to support
+easy registration and execution of a runner container.  They require
+defining several environment variables for use.
+
+#### Runner registration
+
+Each time the registration command is run, a new runner is added into
+the configuration.  If your intent is to simply update or modify the
+configuration, please edit the config.toml file within the
+`gitlab-runner-config` volume.
+
+Note: These commands assume you have both `podman` and `jq` available.
+
+```bash
+$ echo '<registration token>' | podman secret create REGISTRATION_TOKEN -
+$ export IMAGE=<image FQIN:TAG>
+$ eval $(podman inspect --format=json $IMAGE | jq -r .[].Labels.register)
```

Comment:
Is the command right?
```
$ eval $(podman inspect --format=json $IMAGE | jq -r .[].Labels.register)
-bash: null: command not found

$ echo $IMAGE
registry.gitlab.com/rhcontainerbot/pkg-builder:fedora
```

Reply:
@cevich ping

Reply:
The eval command is intended to be used on hosts that don't (yet) support `podman container runlabel` (like CentOS 9 IIRC).  In any case, that's an odd error, it suggests that the image never had the labels added to it, let me check the `Containerfile`...

Reply:
...I copied these details over to gitlab (so I can have automated image builds right now).   This is what I did to get a test-runner up and running for that repo:

```bash
$ echo '<TOKEN>' | podman secret create REGISTRATION_TOKEN -
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel run registry.gitlab.com/qontainers/pipglr:latest
```

Then I check the CI/CD Settings page on gitlab, under runners section, it shows a green dot.

Reply:
Updated docs.